src/chrome/content/rules/Abuse.ch.xml

```diff
@@ -1,14 +1,16 @@
-<ruleset name="Abuse.ch">
-
-	<target host="spyeyetracker.abuse.ch" />
```

Comment:
As long as the hosts is accessible over HTTP we secure it. Otherwise you could still do nasty things with it.